test/setup.ts

```diff
@@ -2,7 +2,7 @@
 
 process.env.LOG_LEVEL = 'fatal'
 
-const nock = require('nock')
+import nock = require('nock')
```

Comment:
While we're changing this, do you mind going the full-on `import * from` route? It _should_ work just fine:

```suggestion
import nock from 'nock'
```